src/io.jl

```diff
@@ -0,0 +1,130 @@
+# Input/ Output of Unfold results
+"""
+    save_results(results::DataFrame, bids_root::String; 
+        save_folder::String="Unfold",
+        overwrite::Bool=false)
+
+Function to save unfold models in your BIDS root folder. Automatically creates a save_folder (default = "Unfold") in the derivatives and subsequentely safes each model in results according to BIDS.
+
+If overwrite is false the function will check for already saved results in the same folder and with the same name i.e. same subject/session/task/run.
```

Comment:
I would private an example with a final path e.g.

`bids_root/derivatives/Unfold/sub-001/eeg/sub-001_task-bla_unfold.jld2` - or whatever the fileformat

Reply:
general Unfold thought: we should think to compare jld2 vs. hdf5 or apache format, to be better compatible with other libraries maybe...

Reply:
addressed